05-lists-and-dictionaries/capital_city_loop.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from capitals import capitals_dict
+from random import choice
+
+def get_random_state():
+    return choice(list(capitals_dict.keys()))
+
+def capital_of_state_quiz(state):
+    while True:
+        response = input("What is the capital of " + state + "? ").lower()
+
+        if check_answer(response, capitals_dict[state]):
+            print("Correct")
+            return True
+        elif response == "exit":
+            print("The correct answer is: " + capitals_dict[state])
+            print("Goodbye")
+            return True
+        else:
+            print("Wrong - try again or type exit to quit")
+
+def check_answer(answer, correct_answer):
+    if answer.lower() == correct_answer.lower():
```

Comment:
Instead of checking if something is True and then return True, otherwise return False, you can just return the evaluation, `return answer.lower() == correct_answer.lower()`.